pom.xml

```diff
@@ -1108,6 +1138,19 @@
       <properties>
         <java.version>8</java.version>
       </properties>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-compiler-plugin</artifactId>
+            <configuration>
+              <compilerArgs combine.children="append">
+                <arg>-J-Xbootclasspath/p:${settings.localRepository}/com/google/errorprone/javac/${error-prone.jdk8.workaround.javac.version}/javac-${error-prone.jdk8.workaround.javac.version}.jar</arg>
```

Comment:
would you mind adding the following link as a comment here?

https://github.com/google/error-prone/blob/f8e33bc460be82ab22256a7ef8b979d7a2cacaba/docs/installation.md#jdk-8


BTW, I think `error-prone.jdk8.javac.version` is clear enough